src/test/java/StringTest.java

```diff
@@ -0,0 +1,103 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static constants.AlphabetCharacter.A;
+import static constants.AlphabetCharacter.B;
+import static constants.AlphabetCharacter.C;
+import static constants.NumberCharacter.ONE;
+import static constants.NumberCharacter.TWO;
+import static constants.SpecialCharacter.CLOSE_PARENTHESIS;
+import static constants.SpecialCharacter.COMMA;
+import static constants.SpecialCharacter.OPEN_PARENTHESIS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.SoftAssertions.assertSoftly;
+
+@DisplayName("String")
+class StringTest {
+
+    private static final String NUMBERS_WITH_COMMA = ONE + COMMA + TWO;
+    private static final String ALPHABETS = String.valueOf(A) + B + C;
+
+    @Nested
+    @DisplayName("split()")
+    class SplitTest {
+
+        @Test
+        @DisplayName("\"1,2\" 문자열을 \",\" 구분자로 하여 split 함수를 사용하면 1과 2로 분리된다.")
+        void test01() {
+            // when
+            final var numbers = NUMBERS_WITH_COMMA.split(COMMA);
```

Comment:
```suggestion
            final var numbers = "1,2".split(",");
```

테스트 코드의 상수 값을 모두 상수화하는 것은 오히려 가독성을 떨어트리는 것은 아닐까?
위와 같이 해당 테스트 코드 내에서 값과 코드의 의미를 모두 파악 가능하도록 하는 것은 어떨까?